man/crystal.1

```diff
@@ -0,0 +1,259 @@
+.\"Crystal Programming Language
+.Dd February 11, 2016
```

Comment:
I tried to remove it, and all the formatting went away. I can try to remove it and find a way to make it work, if you want.